bin/partner-provision.sh

```diff
@@ -11,7 +11,7 @@ SCRIPT_DIR=$(CDPATH='' cd -- "$(dirname -- "$0")" && pwd -P)
 cd "$SCRIPT_DIR" || exit
 
 usage () {
-    echo "Usage: partner-provision.sh --partner_id=partner_id --partner_secret=partner_secret [--user=wp_user_id] [--plan=plan_name] [--wpcom_user_id=1234] [--url=http://example.com]"
+    echo "Usage: partner-provision.sh --partner_id=partner_id --partner_secret=partner_secret [--user=wp_user_id] [--plan=plan_name] [--onboarding=1] [--wpcom_user_id=1234] [--url=http://example.com]"
```

Comment:
It's just 1 or 0, it seemed the simplest thing to pass. However since currently the default is for onboarding to be disabled, 0 is redundant.

In the future this flag will function more as an override, since I anticipate that onboarding will be switched on/off for all users at the partner or partner-key level.